docs/reference/extensions/gnome-extension.rst

```diff
@@ -0,0 +1,212 @@
+.. _gnome-extension:
+
+GNOME extension
+===============
+
+The GNOME extension, referred to internally as ``gnome``, helps build snaps that use GTK
+3, GNOME 42 and higher, and GLib. This extension provides many of the components needed
+for general desktop apps, making it useful for a broader set of apps outside of those
+tailored for the GNOME desktop.
+
+This extension is compatible with the core22 and core24 bases.
+
+
+.. _gnome-extension-included-plugs:
+
+Included plugs
+--------------
+
+When this extension is used, the following plugs are implicitly opened for the app.
+
+The paths slightly differ between core24 and core22 bases.
+
+.. tabs::
+
+    .. group-tab:: core24
+
+        .. code-block:: yaml
+
+            plugs:
+              desktop:
+                mount-host-font-cache: false
+              gtk-3-themes:
+                interface: content
+                target: $SNAP/data-dir/themes
+                default-provider: gtk-common-themes
+              icon-themes:
+                interface: content
+                target: $SNAP/data-dir/icons
+                default-provider: gtk-common-themes
+              sound-themes:
+                interface: content
+                target: $SNAP/data-dir/sounds
+                default-provider: gtk-common-themes
+              gnome-46-2404:
+                interface: content
+                target: $SNAP/gnome-platform
+                default-provider: gnome-46-2404
+              gpu-2404:
+                interface: content
+                target: $SNAP/gpu-2404
+                default-provider: mesa-2404
+
+    .. group-tab:: core22
+
+        .. code-block:: yaml
+
+            plugs:
+              desktop:
+                  mount-host-font-cache: false
+              gtk-3-themes:
+                  interface: content
+                  target: $SNAP/data-dir/themes
+                  default-provider: gtk-common-themes
+              icon-themes:
+                  interface: content
+                  target: $SNAP/data-dir/icons
+                  default-provider: gtk-common-themes
+              sound-themes:
+                  interface: content
+                  target: $SNAP/data-dir/sounds
+                  default-provider: gtk-common-themes
+              gnome-42-2204:
+                  interface: content
+                  target: $SNAP/gnome-platform
+                  default-provider: gnome-42-2204
+
+.. code-block:: yaml
+
+    plugs:
+      - desktop
+      - desktop-legacy
+      - gsettings
+      - opengl
+      - wayland
+      - x11
+      - mount-observe
+      - calendar-service
```

Comment:
@mr-cal I found this part very confusing in the [original document](https://snapcraft.io/docs/gnome-extension#heading--plugs). I'm not sure what it's trying to suggest – are there two different kinds of plugs? Or was one of these collections meant to be slots?

Reply:
These are the snap-wide plugs.